### DISCUSSION TITLE
Routing traffic

### DISCUSSION THREAD

**Discussion by robflate** [htpcBeginner/docker-traefik#251] [Q&A]
In your old setup, before you moved some containers to the cloud, how did you route traffic to a specific IP in your local network?

What I mean is, I assume you had docker-compose-t2.yml and docker-compose-synology.yml and docker-compose-web.yml each on a different machine on your local network. If they all ran a traefik instance listening on port 80 and 443, how did your router determine which local IP to route traffic to?

If I'm not making sense, a more general question would be;

If I have 3 servers on my local network, each running a traefik instance and some services, how do I setup my routers port forwarding to decide which IP address to route traffic to if it all arrives on port 80 and 443?

Thanks.

**Comment by uasdj25** [comment]
I would think that you would want to configure traefik just on one of the hosts.  You can then configure the other services on other hosts to work with traefik.
